sds_data_manager/sds_data_manager_stack.py

```diff
@@ -31,6 +32,29 @@ def __init__(
             block_public_access=s3.BlockPublicAccess.BLOCK_ALL,
         )
 
+        ########### CONFIG STORAGE
+        # This is the S3 bucket where the configurations will be stored
+        config_bucket = s3.Bucket(
+            self,
+            "CONFIG-BUCKET",
+            bucket_name=f"sds-config-{sds_id}",
+            versioned=True,
+            removal_policy=RemovalPolicy.DESTROY,
+            auto_delete_objects=True,
+            block_public_access=s3.BlockPublicAccess.BLOCK_ALL,
+        )
+
+        s3_deploy.BucketDeployment(
+            self,
+            "DeployConfig",
+            sources=[
+                s3_deploy.Source.asset(
+                    str(pathlib.Path(__file__).parent.joinpath("config").resolve())
```

Comment:
Does this upload everything in the `config` folder, or only things named `config.json`? A comment about that might be good here, I see there were some comments about similar things, so being super clear would help future devs.

Reply:
It uploads any file in `config`. I will add a comment to make that clear.